### ISSUE TITLE
Use a regular combobox for text style selection

### ISSUE THREAD

**Issue by danrabbit** [Philip-Scott/Spice-up#77]
It doesn't look like you can actually type in this box and there is no special markup on the items in the popover so why not just use the native Gtk widget here?

**Comment by aleksandar-stefanovic** [comment]
Can this issue be confirmed or closed?

I think either one of these things should be done:
a) Keep the current style but make the text field uneditable.
b) Change the look to the default combobox.

I can do either, I just need someone to confirm the direction we're going towards.

**Comment by Philip-Scott** [comment]
I think the way to do it would be adding an editable property with it's style class for the widget, and setting it to false on text-style, background-pattern and maybe even on Font once #107 lands :)

What i don't like about comboboxes is how it behaves when you have a long list of items such as that it goes above and bellow the "button" and it wouldn't be consistent between the rest of the "popovers"

**Comment by aleksandar-stefanovic** [comment]
I think that using multiple different widgets wouldn't be too inconsistent, I mean, from my experience with the productivity software, it didn't feel "inconsistent" when different input options and widgets were mixed.

Nonetheless, I will work on this, I will try to make it display a `Gtk.Label`, when it is uneditable. Do you think that setting the `Gtk.Label` is the right way to go? Or should `Entry` be kept, and made uneditable?

**Comment by Philip-Scott** [comment]
Currently the Entry is kept and made uneditable, but maybe it would be best to swap it for a label like you said.

That said, i just remember that i did wanted to add the special markup kinda like how it appears on the Fonts section, but that's definitely for another PR :)

**Comment by aleksandar-stefanovic** [comment]
Oh, I just realized that it is already uneditable, I was confusing it with font picker which is editable, but it also shouldn't be (I may send a PR about that). Personally, it feels good this way, because, even if you click on the entry, because it looks editable, it opens up the list of possible choices, which feels natural.

Well, if we came to a conclusion that we're choosing to go with the custom widget, for the sake of consistency, should this issue be closed?
